compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Dialog.desktop.kt

```diff
@@ -274,10 +274,8 @@ fun Dialog(
 
             update(it)
 
-            if (!it.isDisplayable) {
-                it.makeDisplayable()
-                it.contentPane.paint(it.graphics)
-            }
+            it.forceComposeAndLayout()
```

Comment:
We should call this once before isVisible = true somehow.

Otherwise, we layout/draw every time we change the properties.

Reply:
I already did it differently, and forgot to push.